apps/qubit/modules/search/actions/globalReplaceAction.class.php

```diff
@@ -37,7 +37,7 @@ public function execute($request)
         return;
       }
       // Make sure we have confirmed the action
-      elseif (!isset($request->confirm))
+      if (!isset($request->confirm))
```

Comment:
Haha, I think php-cs-fixer is smarter than I am 😝

Reply:
hahaha